numpy/_core/src/umath/string_buffer.h

```diff
@@ -1325,15 +1333,16 @@ string_lrstrip_chars(Buffer<enc> buf1, Buffer<enc> buf2, Buffer<enc> out, STRIPT
 template <typename char_type>
 static inline npy_intp
 findslice_for_replace(CheckedIndexer<char_type> buf1, npy_intp len1,
-                      CheckedIndexer<char_type> buf2, npy_intp len2)
+                       CheckedIndexer<char_type> buf2, npy_intp len2)
```

Comment:
This was correct before the change.

Reply:
Thank you for pointing that out; I hadn't noticed it before.